deep_qa/data/tokenizers/character_tokenizer.py

```diff
@@ -29,8 +29,8 @@ def index_text(self,
 
     @overrides
     def embed_input(self,
-                    input_layer: 'keras.layers.Layer',
```

Comment:
Oh, yeah, I think I left the `Layer` one like that so that pylint would be faster running on the file in vim (importing keras makes pylint takes several seconds).  But it's fine, I can live with the slowdown.  For `TextDataset`, can we do `import deep_qa.data`, and give the annotation as `'data.TextDataset`?